### PR TITLE
Feat/civ 2739 documentonly [discarded due to git issue]

### PR DESCRIPTION
**Pull request by bhagyashreesharma90**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-2739

### Change description ###
Judge Upload - Document Only Journey

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: bin/build-release-ccd-definition.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ excludeNonProdFiles=${2:-true}
 if [ ${environment} == preview ]; then
    excludedFilenamePatterns="-e *-prod.json,*-SDO-HNL-nonprod.json"
    #for testing HNL uplifting, comment the above line and uncomment below
-   #excludedFilenamePatterns="-e *-prod.json,*-SDO-nonprod.json"
+   #excludedFilenamePatterns="-e *-prod.json,*-SDO-nonprod.json
```
Comment (m-meulendijks-v1) [inline bin/build-release-ccd-definition.sh:11]: Do the parentheses need to be removed?

File: ccd-definition/CaseField/CaseFieldEvidenceUploadJudge-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "Label": " ",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "CaseNoteType",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "documentOnly",
+    "Label": "Upload a document",
+    "HintText": "Each document must be less than 100B, You can upload the following file types: Word, Excel, PowerPoint, PDF, RTF, TXT, CSV, JPG, PNG, BMP, TIF, TIFF",
```
Comment (hmcts-version1-ignacio) [inline ccd-definition/CaseField/CaseFieldEvidenceUploadJudge-nonprod.json:14]: Just a typo, "M" and "." ("100MB.")

Comment (bhagyashreesharma90) [reply ccd-definition/CaseField/CaseFieldEvidenceUploadJudge-nonprod.json]: Corrected.

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Review by miguelMolina3691 — DISMISSED** [review]

**Review by m-meulendijks-v1 — DISMISSED** [review]

**Comment by filippo-possenti-version1** [comment]
This will be closed as github is not picking up new commits for some reason.

A new one has been opened in its place:
https://github.com/hmcts/civil-ccd-definition/pull/1572

**Pull request closed without merging**